doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + op: update - Indicates that an update must be forced
+
++ Request (application/json)
+
+    + Headers
+
+            X-HTTP-Method-Override: PATCH
+
+    + Body
+
+            {
+                "temperature": 21.7,
+                "humidity": 60,
+            }
+
++ Response 200
+
+
+### Remove an entity[DELETE /v2/entities/{entityId}]
+
++ Parameters
+    + entityId: Boe_Idearium (required, string) - Entity ID
+
++ Response 204
+
+### Remove one or more entity attributes[DELETE /v2/entities/{entityId}{?attrs}]
+
++ Parameters
+    + entityId: Boe_Idearium (required, string) - Entity ID
+    + attrs: humidity, seatsNumber - Attributes
+
++ Response 204
+
+## Context Entity Attributes [/v2/entities/{entityId}/attrs]
+
+### List all attributes [GET /v2/entities/{entityId}/attrs]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+
++ Response 200 (application/json)
+
+        {
+            "temperature": 21.7,
+            "humidity": 60,
+            "location": {
+                "value":  "41.3763726, 2.1864475",
+                "type": "geo:point",
+                "crs": "WGS84"
+            }
+        }
+
+### Replace all attributes [PUT /v2/entities/{entityId}/attrs]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+
++ Request (application/json)
+
+        {
+            "temperature": 21.7,
+            "relativeHumidity": {
+                "value": 60,
+                "accuracy": 1
+            },
+            "seatsNumber": 12
+        }
+
++ Response 200
+
+### Get a single attribute value as text [GET /v2/entities/{entityId}/attrs/{attrId}{?options}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute to be retrieved.
+    + options: text (optional, string) - Indicates format as text
+
++ Response 200 (text/plain)
+
+        21.7
+
+### Get a single attribute value as JSON [GET /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute to be retrieved.
+
++ Response 200 (application/json)
+
+        {
+            "value": 21.7
+        }
+
+### Get a single attribute value. Not exists [GET /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: mistake (required, string) - Attribute to be retrieved.
+
++ Response 404 (application/json)
+
+        {
+            "code": 4567,
+            "message": "Not found",
+            "description": "The entity does not have such an attribute"
+        }
+
+### Update an attribute value [PUT /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute ID
```

Comment:
Actually, the right term in NGSI is "name" for attrinutes, not "id". Thus, I'd sugget to change it in the overall document, i.ee. attrId -> attrName.